client/packages/common/src/ui/layout/tables/columns/types.ts

```diff
@@ -75,6 +76,9 @@ export interface Column<T extends RecordWithId> {
   sortInverted: boolean;
   getSortValue?: (row: T) => string | number;
 
+  isError?: (row: T) => boolean;
+  isDisabled?: (row: T) => boolean;
```

Comment:
Added is disabled because I saw this previously: 

https://github.com/openmsupply/open-msupply/blob/9c0c17388ef89fdafcc91b0ecf50d6d19cbf9158/client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditTables.tsx#L131-L136

Which was changed now to just be:

https://github.com/openmsupply/open-msupply/blob/de6113720ff71895d85bab5e585c7a0b1733bede/client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditTables.tsx#L152-L156